src/symbolic_functions.cpp

```diff
@@ -301,12 +303,12 @@ double am_spline(double t, int num, ...) {
     double ss;
     double dudt;
     
-    double ts[num];
-    double us[num];
+    double *ts = new double[num]();;
```

Comment:
If a significant amount of time is lost for (de)allocation here, i would use alloca instead.